### PR TITLE
FHAC-1036: The DTD description in the hibernate configuration files will grab the DTD from the application classpath rather than from the Hibernate website.

### PR DESCRIPTION
**Pull request by danielrf123**
Connect.ear will be able to be deployed to Wildfly and Weblogic offline.

@mhpnguyen, @TabassumJafri assigned to review.

### CODE REVIEW

**Comment by TabassumJafri** [comment]
Are all the hbm files in ConnectCoreLib package only?

**Comment by danielrf123** [comment]
@TabassumJafri There are other HBM files also included in test packages that did not get included in the first commit.  Will include in the next one.

**Comment by mhpnguyen** [comment]
endorse for merging without comments

File: Product/Production/Services/AuditRepositoryCore/src/test/resources/hibernate/auditrepo.hibernate.cfg.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<!DOCTYPE hibernate-configuration PUBLIC "-//Hibernate/Hibernate Configuration DTD//EN" "http://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<!DOCTYPE hibernate-mapping SYSTEM "classpath://org/hibernate/hibernate-configuration-3.0.dtd">
```
Comment (TabassumJafri) [inline Product/Production/Services/AuditRepositoryCore/src/test/resources/hibernate/auditrepo.hibernate.cfg.xml]: Is this hibernate-mapping or hibernate-configuration in DOCTYPE?

File: Product/Production/Common/CONNECTCoreLib/src/test/resources/hibernate/docrepo.hibernate.cfg.xml [hunk 2]
```diff
@@ -1,7 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<!DOCTYPE hibernate-configuration PUBLIC
-"-//Hibernate/Hibernate Configuration DTD//EN"
-"http://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<!DOCTYPE hibernate-mapping SYSTEM "classpath://org/hibernate/hibernate-configuration-3.0.dtd">
```
Comment (TabassumJafri) [inline Product/Production/Common/CONNECTCoreLib/src/test/resources/hibernate/docrepo.hibernate.cfg.xml]: Shouldn't we have hibernate-configuration in DOCTYPE for a cfg file?

File: Product/Production/Services/AuditRepositoryCore/src/test/resources/hibernate/auditrepo.hibernate.cfg.xml (hunk 1 above)
Comment (danielrf123) [inline Product/Production/Services/AuditRepositoryCore/src/test/resources/hibernate/auditrepo.hibernate.cfg.xml]: Yes, I accidentally pasted the "-mapping" DOCTYPE over to the .cfg's in the test package.

File: Product/Production/Common/CONNECTCoreLib/src/test/resources/hibernate/docrepo.hibernate.cfg.xml (hunk 2 above)
Comment (danielrf123) [inline Product/Production/Common/CONNECTCoreLib/src/test/resources/hibernate/docrepo.hibernate.cfg.xml]: Same as above.

**Comment by TabassumJafri** [comment]
Reviewed changes and endorsing PR for merge.

**Pull request merged**